app/views/course_preferences/_form.html.erb

```diff
@@ -10,5 +10,4 @@
   <div class="actions">
     <%= form.submit %>
   </div>
-<% end %>
-<%= javascript_pack_tag 'required_stars' %>
\ No newline at end of file
+<% end %>
```

Comment:
There are several files in this pull request without an empty last line. Please see the red circle icon. If it doesn't bother you you can ignore my comment, but I like to get rid of these red icons in pull requests.
I think it is common practice in Rails and GitHub to end a file with a newline.